api/v1beta1/conditions_const.go

```diff
@@ -187,14 +187,10 @@ const (
 	RebootTimedOutReason = "RebootTimedOut"
 	// CheckDiskFailedReason indicates that checking the health of the disk was not successful.
 	CheckDiskFailedReason = "CheckDiskFailed"
-)
-
-const (
-	// SSHAfterInstallImageSucceededCondition indicates that the host is reachable via ssh after installImage.
-	SSHAfterInstallImageSucceededCondition clusterv1.ConditionType = "SSHAfterInstallImageSucceeded"
-
-	// SSHAfterInstallImageFailedReason indicates that the host was not reachable via ssh.
-	SSHAfterInstallImageFailedReason = "SSHAfterInstallImageFailed"
+	// PreProvisionCommandSucceededReason indicates that the pre-provision command succeeded.
```

Comment:
I don't think you should define this as a reason.. You use it only in an event, where you can hardcode it AFAIK.. All reasons here are for conditions

Reply:
ok